src/humancellatlas/data/metadata/api.py

```diff
@@ -450,6 +450,16 @@ def is_sequencing_process(self):
         return True
 
 
+@dataclass(frozen=True)
+class ImagingTarget:
```

Comment:
Thoughts on the class name? Technically it should be `Target` but it seemed a little too vague.

Reply:
SGTM